app/lib/cooldown.js

```diff
@@ -6,8 +6,15 @@ module.exports = Base => class extends Base {
         
         //object containing the cooldown times for the commands. new commands would need a new property added here.
         this.coodownTimes = {
-            'home': 1000 * 60 * 15,
-            'warp': 1000 * 60 * 15
+            '!home': 1000 * 60 * 15,
+            '!warp': 1000 * 60 * 15,
+            'Location teleporting': 1000 * 60 * 30
+        }
+
+        //allows us to see exactly how much time is left during the cooldown.
+        this.timeLeft = {
```

Comment:
Since we are trying to use classes which means we are attempting Object Oriented Programming. We shouldn't have methods hidden on objects we define within the class. 

You should make 2 methods on the parent class for this.